packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec

```diff
@@ -103,6 +108,8 @@ cp -pa .%{gem_dir}/* \
 %foreman_bundlerd_file
 %foreman_precompile_plugin -a -s
 
+mkdir -p %{buildroot}%{_root_localstatedir}/lib/foreman/%{gem_name}/ansible-playbooks/
```

Comment:
Shouldn't this happen inside smart_proxy_acd instead? As that's the one actually running the playbooks?

Reply:
In %{_root_localstatedir}/lib/foreman/%{gem_name}/ansible-playbooks/ are the ansible playbook stored, which are cloned from git (with the new git support feature).

Reply:
OK